### PR TITLE
fix(auth): resolve issue with omnect-cli freezes

### PR DESCRIPTION
**Pull request by empwilli**

### CODE REVIEW

File: src/auth.rs [hunk 1]
```diff
@@ -152,19 +160,29 @@ async fn request_access_token(auth_info: &AuthInfo) -> Result<Token> {
...
+    }
 
     log::info!("Redirecting to authentication provider.");
     log::info!(
         "Note: if the browser does not open automatically, use this link to complete login: {}",
         auth_url.to_string()
     );
-    let _ = open::that(auth_url.to_string());
+
+    // let _ = open::that(auth_url.to_string());
```
Comment (JoergZeidler) [inline src/auth.rs:185]: could be deleted

File: src/auth.rs [hunk 2]
```diff
@@ -215,6 +233,8 @@ where
         request_access_token(&auth_info).await?
     };
 
+    println!("token: {:#?}", token.access_token().secret());
```
Comment (JoergZeidler) [inline src/auth.rs:236]: just for debugging, could be deleted

File: Cargo.toml [hunk 3]
```diff
@@ -2,12 +2,10 @@
 authors = ["omnect@conplement.de>"]
 description = "This tool provides a command-line interface to configure omnect-os images and to communicate with omnect-os empowered devices."
 edition = "2021"
-homepage = "https://www.omnect.io/home"
```
Comment (JoergZeidler) [inline Cargo.toml:5]: why it's removed?

File: Cargo.toml [hunk 4]
```diff
@@ -2,12 +2,10 @@
...
-license = "MIT OR Apache-2.0"
```
Comment (JanZachmann) [inline Cargo.toml:6]: same as above

File: Cargo.toml (hunk 3 above)
Comment (empwilli) [reply Cargo.toml:5]: see below.

**Review by JoergZeidler — APPROVED** [review]

**Review by JanZachmann — APPROVED** [review]

**Pull request merged**